### ISSUE TITLE
Main volume is updated on playing start

### ISSUE THREAD

**Issue by frague59** [hslbck/gnome-shell-extension-radio#110]
Hi, 
I'm using gnome radio for a long time, and it's quite a very good extension: it has became the main radio in my home...The PC is plugged into the main HiFi, it's cool. 

When using a header set at work, I can notice that the main (aka. pulse audio) playing volume is pumped up to max, which destroys my ears every time I start listening a channel or change it.

Could this be fixed ?
Thanks for this great extension !

**Comment by hslbck** [comment]
Hi,
do you use the separate volume control for the extension, enabled via settings, or do you use the default volume provided by gnome-shell?
Do you plugin the headset everytime you start listening to music? I keep the headset plugged in and am not facing the issue.

BR

**Comment by frague59** [comment]
Hi,
I'm using the default volume controller provided by gnome-shell : I didn't notice I could have a volume controler inside the applet...

I did just the test: I'm listening FIP Rock, a (good) french rock radio, at about 20% volume, and then I swap to another channel: the volume is back to 100%. I did not unplug the headset. 

I've tested with the applet volume, I've enabled it, tuned down the volume, and then disabled it: the sound stays at the previously set volume : 20% in applet, 100% in gnome.

Ive tryed to disable and then re-enable the applet : sound says at 20% in my hears, 20% in the applet, and 100% in gnome.

Thanks !
